packages/react/src/components/InAppMessaging/index.ts

```diff
@@ -1,3 +1,4 @@
+export { Backdrop } from './Backdrop';
```

Comment:
I don't think this should be exported out of this folder, but just made available to to use internally within the InAppMessaging UI components. By exposing it this way it may get accidentally exported as a public API before we are ready. It should be internal only for now.

Reply:
Good call will do 👍